src/com/mindscriptact/mvcExpressLogger/screens/MvcExpressLogScreen.as

```diff
@@ -5,7 +5,7 @@ import flash.display.Sprite;
 
 /**
  * COMMENT
- * @author Raimundas Banevicius (http://mvcexpress.org/)
```

Comment:
are you sure you had the latest version? this change looks weird.. from old file version.